### ISSUE TITLE
[Elasticsearch] wrong metrics port

### ISSUE THREAD

**Issue by dis-sid** [bitnami/charts#4448]
**Which chart**:  elasticsearch-12.8.0

**Describe the bug**
I run a GKE cluster with BKPR components installed on it.
When deploying the chart using values-production.yaml, prometheus shows one target down:
`Get "http://a.b.c.d:8080/metrics": dial tcp a.b.c.d:8080: connect: connection refused`

It's the endpoint from elasticsearch-metrics pods. I see that provided values.yaml files are setting annotation "prometheus.io/port" to "8080" although pods are opening http port 9114.

**To Reproduce**

1. Deploy the bitnami elastic helm chart on a kubernetes cluster with prometheus from BKPR 
2. Look at prometheus /targets path.

**Expected behavior**
No Prometheus target down.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.0", GitCommit:"7090a89efc8a18f3d8178bf47d2462450349a004", GitTreeState:"clean", GoVersion:"go1.15.3"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"archive", BuildDate:"2020-10-16T20:58:56Z", GoVersion:"go1.15.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.13-gke.404", GitCommit:"81e2afbf51b876d15f0af546998c670207fdd592", GitTreeState:"clean", BuildDate:"2020-10-30T23:52:44Z", GoVersion:"go1.13.9b4", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by miguelaeh** [comment]
Hi @dis-sid ,
Could you check if changing the line `prometheus.io/port: "8080"` inside the metrics section of the values.yaml to `prometheus.io/port: "9114"` fixes it?

**Comment by dis-sid** [comment]
@miguelaeh Hi. Looking at the helm values file and the kubernetes services created I inferred that `metrics.service` from the values is used to configure a k8s service which is targeting the misannotated pods. So I've just removed pods annotations. Since I'm making a demo of the deployed monitoring/logging infrastructure to my colleagues today I won't try to get pods annotations back with port 9114. But I can easily forecast that it would work fine since the kubernetes service is already targeting pods on port 9114 and the service seems to work well. I would recommend changing the values.yaml files, removing pods annotation (or setting port to 9114) and close the issue.
